src/apis/mergeDocument.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Merge a file into the currently opened document
+ * @method WebViewer#mergeDocument
+ * @param {(string|File)} documentPath Path to the document OR <a href='https://developer.mozilla.org/en-US/docs/Web/API/File' target='_blank'>File object</a> if opening local file.
+ * @param {number} position Position to put the merged document, default to end of file if nothing entered
+ * @returns {Promise<any>} a promise that resolves on completion
+ * @example // 5.1 and after
+WebViewer(...)
+  .then(function(instance) {
+    instance.mergeDocument('https://www.pdftron.com/downloads/pl/test.pdf', 1);
+  });
+ */
+import actions from 'actions';
+
+export default store => (documentToMerge, position) => {
```

Comment:
In the ThumbnailPanel, I need it to allow users to drag and drop file to merge.

I can provide you with more information on customer requirements  (not sure if I should post it here since it's public)